miqa/core/tasks.py

```diff
@@ -49,6 +49,21 @@ def _download_from_s3(path: str, public: bool) -> bytes:
     return buf.getvalue()
 
 
+@shared_task
+def reset_demo():
+    if not settings.DEMO_MODE:
+        raise Exception('Demo Mode is false; noop.')
```

Comment:
```suggestion
    if not settings.DEMO_MODE:
        raise RuntimeError('Demo Mode is false; noop.')
```

If you want to keep this, consider [`RuntimeError`](https://docs.python.org/3/library/exceptions.html#RuntimeError), which is a slightly more specific type.

Reply:
Removed in [418868c](https://github.com/OpenImaging/miqa/pull/635/commits/418868c2527eca6e3bbd4f287477a48908834cce)